lib/brook/server.ex

```diff
@@ -38,6 +40,24 @@ defmodule Brook.Server do
   end
 
   def handle_call({:process, %Brook.Event{} = event}, _from, state) do
+    process(event, state)
+    {:reply, :ok, state}
+  end
+
+  def handle_call({:send, type, author, event}, _from, state) do
+    :ok = apply(state.driver.module, :send_event, [type, author, event])
+    {:reply, :ok, state}
+  end
+
+  def handle_cast({:process, %Brook.Event{} = event}, state) do
+    process(event, state)
+    {:noreply, state}
+  end
+
+  defp process(%{forwarded: false} = event, state) do
+    IO.inspect(event, label: "Origin event")
```

Comment:
Pull this out or change it to a `Logger.debug` or `Logger.info` statement?

Reply:
fixed